cob_hardware_config/cob4-3/config/base_joint_limits.yaml

```diff
@@ -0,0 +1,73 @@
+joint_limits:
+  fl_caster_rotation_joint:
+    has_position_limits: false
+    min_position: -3.14
+    max_position: 3.14
+    has_velocity_limits: true
+    max_velocity: 15.0
+    has_acceleration_limits: false
+    max_acceleration: 2.0
+    has_jerk_limits: false
+    max_jerk: 100.0
+    has_effort_limits: false
+    max_effort: 5.0
```

Comment:
I would suggest to keep only velocities and acceleration limits.
Acceleration might be supported in the near future..
For now they can be commented out.

There is no need for position, effort and jerk limits.